app/src/main/java/com/hedvig/app/feature/profile/ui/payment/PaymentHistoryAdapter.kt

```diff
@@ -17,6 +18,7 @@ class PaymentHistoryAdapter(
     override fun getItemViewType(position: Int) = when (items[position]) {
         is ChargeWrapper.Header -> HEADER
         is ChargeWrapper.Item -> ITEM
+        is ChargeWrapper.Title -> TITLE
```

Comment:
More optional improvements: Make this `RecyclerView.Adapter` follow all our more recent patterns 🙂